src/app/pages/assignable/assignable.scss

```diff
@@ -0,0 +1,56 @@
+@import 'pattern-library/core/pattern-library/headers';
+
+$near-white: #f8f8f8;
+
+.assignable.page {
+
+    @include set-font(body-large);
+
+    section {
+        @include scale-feature(padding-top, 2rem, 12rem);
+        @include scale-feature(padding-bottom, 2rem, 12rem);
+
+        max-width: 100%;
+        overflow: auto;
+    }
+
+    .content-block {
+        @extend %content;
+        @include scale-feature(grid-gap, $normal-margin, 4rem);
+
+        display: grid;
+
+        > * {
+            margin: 0;
+        }
+    }
+
+    .btn.primary {
+        @extend %button;
+        @extend %primary;
+    }
+
+    .near-white {
```

Comment:
just a nitpick but are these colors available in some more global-esque css file?

Reply:
Reasonable point. We have something that's a little less white, which I think improves contrast. I'll use that.